Library/Homebrew/cmd/commands.rb

```diff
@@ -8,6 +8,14 @@ module Homebrew
   module_function
 
   def commands
+    require "options_for_brew_commands"
+    options do
+      command __method__.to_s
```

Comment:
This `__method__.to_s` is ugly; I wonder if there’s a clean way to do that without changing a lot of things in the way we handle commands.